cmp/dataview/DataViewModel.js

```diff
@@ -36,17 +40,20 @@ export class DataViewModel {
      * @param {(Object[]|GridStoreContextMenuFn)} [c.contextMenu] - array of RecordActions, configs or token
      *      strings with which to create grid context menu items.  May also be specified as a
      *      function returning a StoreContextMenu.  Desktop only.
+     * @param {number} itemHeight - Row height for each item displayed in the view
```

Comment:
Can be order this in the comment and constructor params below to come after the itemRenderer? We usually put required params first, before the optional ones